backend/event/types/EpisodeAgentType.py

```diff
@@ -1,24 +1,23 @@
-from graphene import ResolveInfo
+from graphene import NonNull
 from graphene_django import DjangoObjectType
-from core.types.DescriptionFieldType import DescriptionFieldType
-from django.db.models import QuerySet
-
-from event.models import EpisodeGift
 
+from core.types.DescriptionFieldType import DescriptionFieldType
+from event.models import EpisodeAgent
+from core.types.entity import Entity, EntityDescription
+from event.types.EpisodeEntityLink import EpisodeEntityLink
+from core.types.DescriptionFieldType import SourceMentionEnum
 
 class EpisodeAgentType(DescriptionFieldType, DjangoObjectType):
+    entity = NonNull(EntityDescription)
+    entity_type = NonNull(Entity)
+
     class Meta:
-        model = EpisodeGift
+        model = EpisodeAgent
         fields = [
             "id",
             "episode",
             "agent",
+            "entity",
```

Comment:
"entity" and "entity_type" do not exist on the model, so they don't belong here.